test/service/test_cart_export_endpoints.py

```diff
@@ -46,9 +46,9 @@ def test_unified_endpoint_initiate_ok(self):
                     response = requests.post(export_url,
                                              headers=dict(Authorization=f'Bearer {test_jwt}'),
                                              allow_redirects=False)
-        self.assertEquals(301, response.status_code)
-        self.assertEquals(f'{export_url}?token={expected_export_token}',
-                          response.headers['Location'])
+        self.assertEqual(301, response.status_code)
```

Comment:
Now that you went there, these cosmetic changes should be in a separate commit.

Reply:
@hannes-ucsc these changes were done  in bab2df07001bb31eb4dc7e54ac5d198524d332e9 do i need to change something?

Reply:
No, I missed you already had a separate commit for this.